### PR TITLE
use absolute path for walking

### PR DESCRIPTION
**Pull request by redonkulus**
@jedireza

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
  * Copyright 2014, Yahoo! Inc.
  * Copyrights licensed under the New BSD License. See the accompanying LICENSE file for terms.
  */
+process.chdir(__dirname);
```
Comment (jedireza) [inline server.js]: we should add a comment about why we do this

File: services/docs.js [hunk 2]
```diff
@@ -4,14 +4,17 @@
...
 'use strict';
 
+var path = require('path');
 var debug = require('debug')('DocsService');
 var fs = require('fs');
 var walk = require('walk');
 var marked = require('marked');
 var highlight = require('highlight.js');
 
+var CWD = process.cwd();
```
Comment (jedireza) [inline services/docs.js]: Let's just use `process.cwd()` instead of creating a variable for it.

Comment (redonkulus) [inline services/docs.js]: i saved it cause its used in the file event.

**Comment by jedireza** [comment]
:+1:

**Pull request merged**